tests/test_exposureTable.py

```diff
@@ -189,6 +192,8 @@ def testPersistence(self):
             self.assertIsNone(cat1[1].getVisitInfo())
             self.assertIsNone(cat1[0].getDetector())
             self.assertDetectorsEqual(cat1[1].getDetector(), self.detector)
+            for key in self.plist.keys():
```

Comment:
Don't need the `keys()` here.  Also please add a comment that you are doing it this way rather than direct equality because the property list read from the file will include extra keys.